src/controller.ts

```diff
@@ -112,6 +112,10 @@ export class AugurNodeController {
     this.logger.addLogger(logger);
   }
 
+  public clearLoggers() {
+    this.logger.clear();
```

Comment:
Should that be `this.logger.clearLoggers()`?

Reply:
.clear method already existed so gtg.